quickstart/101-azfw-with-fwpolicy/variables.tf

```diff
@@ -0,0 +1,18 @@
+// Create Variables for Location and Tags
+variable "location" {
+  default = "eastus"
+}
+variable "tags" {
+  default = {
+    environment = "dev"
+    costcenter  = "1234556677"
+    owner       = "cloud team"
+    workload    = "azure firewall"
+  }
+}
+
+// Create Firewall Variables
+variable "fw_sku" {
```

Comment:
Please update this to use the appropriate validation block. The following sample has a couple of examples (`app_service_plan_sku_name`, `front_door_sku_name`): https://github.com/Azure/terraform/blob/master/quickstart/101-front-door-standard-premium/variables.tf